.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!-- Please see https://github.com/zulip/zulip-terminal#contributor-guidelines ! -->
+
+**What does this PR do?**
+
+Overall description goes here!
+
+- [ ] New Feature?
+- [ ] Fixes Bug? [Fixes <issue>; Partially <issue> | Fixes point <n> of <issue> | Resolves an issue that's not filed]
```

Comment:
Perhaps also include the discussion link, if there's no issue filed for the fix?

Reply:
Good point, though I think that's a separate entry to suggest including.